### DISCUSSION TITLE
unsuppoorted MSH versiuon 4.1

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#564] [Q&A] [answered]
why this problem happens? and where is the file tpv105_half.xdmf
![image](https://user-images.githubusercontent.com/104721358/171588669-2a9ab4c1-4e63-4740-809e-512ab942c8d8.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
fixed by https://github.com/SeisSol/Examples/commit/065f99c9338eb515671f517dcb9a5b045c4a0e90
